phpcs.xml.dist

```diff
@@ -79,6 +79,9 @@
 
 		<!-- Allow short array syntax -->
 		<exclude name="Generic.Arrays.DisallowShortArraySyntax.Found" />
+
+		<!-- Do not enforce 'class-' prefix -->
+		<exclude name="WordPress.Files.FileName.InvalidClassFileName" />
```

Comment:
Are we sure about adding this rule?

Reply:
@pauarge, please see this thread: https://github.com/Automattic/vip-go-mu-plugins/pull/2429#pullrequestreview-755683336